src/Page.re

```diff
@@ -250,7 +250,7 @@ external emulateMediaDisable:
 /** Iterates the JS heap finding all the objects with the given prototype. */
 [@bs.send]
 external queryObjects:
-  (t, ~prototypeHandle: JSHandle.t) => Js.Promise.t(JSHandle.t) =
+  (t, ~prototypeHandle: JSHandle.t('a)) => Js.Promise.t(JSHandle.t('b)) =
```

Comment:
It's kind of weird that there are two nearly identical `queryObjects` functions defined in different classes, but this was how Puppeteer did it. We should make sure they are typed consistently (return type doesn't match the other one currently).